posthog/warehouse/models/datawarehouse_saved_query_draft.py

```diff
@@ -11,7 +11,7 @@ class DataWarehouseSavedQueryDraft(CreatedMetaFields, UpdatedMetaFields, UUIDMod
     """
 
     team = models.ForeignKey(Team, on_delete=models.CASCADE)
-    query = models.JSONField(default=dict, null=True, blank=True, help_text="HogQL query draft")
+    query = models.JSONField(default=dict, blank=True, help_text="HogQL query draft")
```

Comment:
Think we need to remove `blank=true` too - looks like the migration itself still has nullable set

Reply:
I just didn't update the migration it should work now here: https://github.com/PostHog/posthog/pull/35978/